### ISSUE TITLE
Node appears to be incorrectly mapped to Biolink category

### ISSUE THREAD

**Issue by karafecho** [RobokopU24/qgraph#258]
This issue is to report that a node representing "aspirin-induced asthma" is incorrectly mapped to either `biolink:LifeStage` or `biolink:PhenotypicFeature`, which was made apparent in the question-builder by a distinct color.

[See correct screenshot below.]

**Comment by Woozl** [comment]
@EvanDietzMorris Kara found this `HP:0012042` (aspirin-induced asthma) with the category `biolink:PhenotypicFeature`. Which is different from most of the other "asthma" nodes which have the `biolink:Disease` category. Is this a "bug" that needs to be fixed or do we just accept a few quirks like this in the knowledge graph? 

![image](https://github.com/RobokopU24/qgraph/assets/16181779/f9aa789d-9c47-461b-a183-1cc0cb409b60)

<details>
<summary>HP:0012042 JSON</summary>

```json
"HP:0012042": {
  "categories": [
    "biolink:ThingWithTaxon",
    "biolink:PhenotypicFeature",
    "biolink:DiseaseOrPhenotypicFeature",
    "biolink:BiologicalEntity",
    "biolink:NamedThing",
    "biolink:Entity"
  ],
  "name": "Aspirin-induced asthma",
  "attributes": [
    {
      "attribute_type_id": "biolink:same_as",
      "value": [
        "SNOMEDCT:407674008",
        "UMLS:C1319853",
        "MESH:D055963",
        "HP:0012042"
      ],
      "value_type_id": "metatype:uriorcurie",
      "original_attribute_name": "equivalent_identifiers"
    },
    {
      "attribute_type_id": "biolink:has_count",
      "value": 1905,
      "value_type_id": "EDAM:data_0006",
      "original_attribute_name": "omnicorp_article_count"
    },
    {
      "attribute_type_id": "dct:description",
      "value": "A type of asthma in which aspirin and other nonsteroidal anti-inflammatory drugs (NSAIDs) that inhibit cyclooxygen-ase 1 (COX-1) exacerbate bronchoconstriction.",
      "value_type_id": "EDAM:data_0006",
      "original_attribute_name": "description"
    }
  ]
}
```

</details>

**Comment by EvanDietzMorris** [comment]
The assignment of node categories really comes from the node normalizer so any discussion of whether we should change it should happen there. See: https://nodenormalization-sri.renci.org/1.3/get_normalized_nodes?curie=HP%3A0012042&conflate=true

That being said, HP is the Human Phenotype Ontology so that's why it was assigned that. More importantly, this is a good case of why we have the DiseaseOrPhenotypicFeature type and it is assigned that. For a lot of queries we should probably be using that instead, conflating diseases and phenotypes and giving both back as answers.

**Comment by cbizon** [comment]
I'm not sure what the issue is here.  @karafecho?  Is there a MONDO term or some other disease vocabulary with this term?

**Comment by karafecho** [comment]
I think there were a few things bundled into this issue.

1. At the time I posted the issue, there wasn't a legend, so while I could see that the color was different for "aspirin-induced asthma", I couldn't tell what the Biolink category was. It looks like David has since added a legend (thank you! that was fast!), so that issue is resolved.
2. The other issue is that users cannot see the CURIE, in this case HP:0012042, or the Biolink category without digging through the result JSON, which I frankly didn't even think of at the time.
3. Related to (2), I think it would be useful for users to be able to click or hover over a node and see the CURIE and/or the metadata, for instance, https://hpo.jax.org/app/browse/term/HP:0012042. If I recall, the prior version of ROBOKOP supported this feature. Might be good to add this back. I'll post a ticket.
4. The other thing that's seems weird to me, is that there isn't a MONDO for "aspirin-induced asthma", although it looks like "aspirin-induced asthma" is a synonym of "asthma, nasal polyps, and aspirin intolerance": https://monarchinitiative.org/disease/MONDO:0008834. Not sure if this is worth addressing.

If (4) doesn't seem worth addressing, then this issue can be closed.

**Comment by cbizon** [comment]
I wouldn't say "not worth addressing", but it'd be tough.  We (mostly) trust MONDO and HP on what's a disease and what's a phenotype.  There are a few where we merge them, but even there we really need to clean things up and consider that a conflation.  This one is even trickier, bc neither MONDO nor HP references the other, and their common xrefs differ as well.  So I'd at least say out of scope for robokop. 

Should we make issues for 1-3?
